app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -121,30 +146,7 @@
       android:layout_width="match_parent"
       android:layout_height="match_parent" />
 
-    <FrameLayout
-      android:id="@+id/hints_and_solution_fragment_container"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_gravity="bottom|start"
-      android:background="@drawable/hints_background"
-      android:visibility="@{viewModel.isHintBulbVisible() ? View.VISIBLE : View.GONE}">
-
-      <ImageView
-        android:id="@+id/dot_hint"
-        android:layout_width="8dp"
-        android:layout_height="6dp"
-        android:layout_gravity="top|end"
-        android:layout_margin="8dp"
-        android:src="@drawable/ic_dot_yellow_24dp"
-        android:visibility="@{viewModel.isHintOpenedAndUnRevealed() ? View.VISIBLE : View.GONE}" />
-
-      <ImageView
-        android:id="@+id/hint_bulb"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_margin="12dp"
-        android:src="@drawable/ic_hint_bulb_white_24dp" />
-    </FrameLayout>
+
 
```

Comment:
This entire file needs reformatting.

Reply:
@aishapervin You need to press `Ctrl + Shift + Alt + L` to auto-format this complete file at once so that we can get rid of empty lines and other formatting errors.

Reply:
done